SEED-platform/seed
# 2330/fix/unitted mapping results

#### What's this PR do? 
- Fixes sorting of unitted columns on mapping page. Specifically, it fixes `import_files/{id}/mapping_results` so that it returns numbers rather than strings for any pint values (e.g. eui or area).
- Fixes displaying correct number of decimals for unitted columns. Specifically, fixes inventory list and mapping controllers for displaying proper number of decimal places for eui and area values according to organization settings.
#### How should this be manually tested?
- Upload taxlot/property and choose Site EUI and an area field for a column, on the mapping page it should show the correct number of decimal places. Sorting the columns should work as well
- On inventory list page, any eui or area columns should have correct number of decimal places. Should be able to sort these columns
#### What are the relevant tickets?
#2330
